### ISSUE TITLE
Isn't "Mockingboard C audio (without SSI-263 voice)" just a Mockingboard A?

### ISSUE THREAD

**Issue by IvanExpert** [samkusin/clemens_iigs#150]
Just curious, because it is confusing when reading the instructions -- when I saw that Mockingboard C was indicated, I assumed that necessarily implied speech, as the C was an A card with the "Mockingboard B" speech chip pre-installed.

Thanks for clarifying!

**Comment by samkusin** [comment]
Yeah.   The original plan was to add the SSI component while I was implementing this but didn't get to it.   So I basically added that note without reviewing it because yes you are correct.   I'll change the README/docs to reflect this once I start working on the emulator again.

**Comment by IvanExpert** [comment]
Ah, nice, thanks. Just wanted to make sure my understanding is correct. Looking forward to the next rev of Clemens, we need more IIgs emulators!
